cookbook/entities/acl.rst

```diff
@@ -0,0 +1,296 @@
+How to Protect Entities Using ACLs
```

Comment:
Why this file exist in this PR? The newest version already exist at https://github.com/orocrm/documentation/pull/206/files#diff-f924240c2ac5eca3ac8df47bd7e79914R1